n3fit/src/n3fit/hyper_optimization/penalties.py

```diff
@@ -82,15 +82,15 @@ def patience(stopping_object=None, alpha=1e-4, **_kwargs):
     >>> from n3fit.hyper_optimization.penalties import patience
     >>> from types import SimpleNamespace
     >>> fake_stopping = SimpleNamespace(e_best_chi2=1000, stopping_patience=500, total_epochs=5000, vl_loss=2.42)
-    >>> patience(None, fake_stopping, alpha=1e-4)
+    >>> patience(fake_stopping, alpha=1e-4)
     3.434143467595683
 
     """
     epoch_best = np.take(stopping_object.e_best_chi2, 0)
     patience = stopping_object.stopping_patience
     max_epochs = stopping_object.total_epochs
     diff = abs(max_epochs - patience - epoch_best)
-    vl_loss = np.take(stopping_object.vl_chi2, 0)
+    vl_loss = np.take(stopping_object.vl_loss, 0)
```

Comment:
```suggestion
    vl_loss = np.take(stopping_object.vl_chi2, 0)
```

I see. Then let me revert that.